test/analytics/analytics-on-redshift/workflow/load-data-stepfuncs.test.ts

```diff
@@ -38,5 +39,19 @@ describe('Load data workflow', () => {
     expect(JSON.stringify(strCapture.asObject(), undefined, 2)).toEqual(loadDataStepFuncDef);
   });
 
```

Comment:
add test case to Verify the sub-workflow is invoked with proper properties to map the tables